### PR TITLE
Commerce bom

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+1. Enter a name for the product (Generic Diesel Engine).
+1. Click _Save_.
+
+## Creating a Definition
+
+1. Click the _Entries_ tab.
+
+    ![BOM Entries tab](./managing-boms/images/02.png)
+
+1. The _Mapped Products_ pane on the right lists all subparts. WHen users hover the mouse over the diagram, a plus sign appears.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md:23]: > WHen

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    ![BOM Entries tab](./managing-boms/images/03.png)
+
+1. Click on the part of the diagram that is associated wih product (for example, _Fuel Injector 1_).
```
Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md:27]: > associated wih product

**Comment by jrhoun** [comment]
I was going to continue reviewing the article with the first typo I found, but then I found a second and decided to send it back. Please re-review the entire article and re-send. Thanks!

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md:23]: Removed extra capital

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/managing-boms.md:27]: fixed typo

**Pull request closed without merging**